### PR TITLE
Nexpose parser: Add "Host Up" + "Open port" and handling non-standard names of services

### PR DESCRIPTION
**Pull request by kiblik**
- Inspired by NMAP parser
- add non-standard service names to nexpose reports
- cleaning service names and tags

### CODE REVIEW

File: dojo/tools/nexpose/parser.py [hunk 1]
```diff
@@ -184,6 +185,14 @@ def get_items(self, tree, vulns, test):
                 host['services'] = list()
                 host['vulns'] = self.parse_tests_type(node, vulns)
 
+                host['vulns'].append({
+                    'name': 'Host Up',
+                    'desc': 'Host is up because it replied on ICMP request or some TCP/UDP port is up',
+                    'severity': 'Info',
+                    'resolution': 'N/A',
+                    'vector': 'No impact provided',
```
Comment (damiencarol) [inline dojo/tools/nexpose/parser.py:193]: Not fan of 'N/A' and 'No xxx provided'. We have null-able attributes so, we can find a way to not set them if there is no data.

Comment (kiblik) [reply dojo/tools/nexpose/parser.py]: Fixed https://github.com/DefectDojo/django-DefectDojo/pull/4564/commits/506033e219c51e6e3fa15ae64ce2124fc59f1a34

File: dojo/unittests/tools/test_nexpose_parser.py [hunk 2]
```diff
@@ -109,8 +186,8 @@ def test_nexpose_parser_dns(self):
         # self.assertEqual('dns://192.168.1.1#tcp', str(finding.unsaved_endpoints[0]))
         # self.assertEqual('dns://192.168.1.1#udp', str(finding.unsaved_endpoints[1]))
```
Comment (damiencarol) [inline dojo/unittests/tools/test_nexpose_parser.py:187]: I think you can implement these two

**Review by damiencarol — APPROVED** [review]
It's a good addon

File: dojo/unittests/tools/test_nexpose_parser.py (hunk 2 above)
Comment (kiblik) [reply dojo/unittests/tools/test_nexpose_parser.py]: You are right :) I forgot

**Review by valentijnscholten — APPROVED** [review]

**Pull request merged**